scripts/set_tenant_library.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -Eeuo pipefail
+
+# help function used when input parameters are empty
+help_function()
+{
+    echo ""
+    echo "Usage : ./set_library_tenant <owner_key> <comma separated libraries list> <tenant_contract_id>"
```

Comment:
Can we require $PRIVATE_KEY?  That way we don't put keys as command argument

Reply:
made the changes